### PR TITLE
Homework part/1

### PR DESCRIPTION
**Pull request by ghifo**

### CODE REVIEW

File: Week2/homework/part1/homework.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  try {
+    console.log(questions);
+    const question = await input('number');
+    const questionNumber = question['number'];
+
+    let userInput;
+    let result;
+    let Region;
+    let Language;
+
```
Comment (ufukcalimli) [inline Week2/homework/part1/homework.js]: Region and Language variables should be lowercase

File: Week2/homework/part2.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  DECLARE language_Number INT;
+  SET language_Number = (SELECT COUNT(Language) FROM countrylanguage JOIN country ON Code = Country_Code WHERE code = new.Country_Code);
+  IF language_Number >= 10
+  THEN
+    SET message = 'Alert: no more languges can be added to the country!';
+    SET lc_messages=message; SIGNAL SQLSTATE '45000';
+  END IF;
+END $$
+DELIMITER ;
+
```
Comment (ufukcalimli) [inline Week2/homework/part2.sql]: Good job!

**Comment by fedefu** [comment]
Thanks for the feedback @ufukcalimli. Can you please update your code @ghifo? cheers

**Comment by ghifo** [comment]
Sure I will

File: Week2/homework/part1/homework.js (hunk 1 above)
Comment (ghifo) [reply Week2/homework/part1/homework.js]: I did not notice that, thanks!

**Comment by fedefu** [comment]
@ufukcalimli thanks for all the feedback! Can you please do a final review so we close this PR? Cheers

**Comment by ghifo** [comment]
I did it long ago 🤔

**Pull request closed without merging**